### ISSUE TITLE
Don't downgrade pytorch when installing?

### ISSUE THREAD

**Issue by ducha-aiki** [naver/kapture#7]
Hi,

First, thank you for the great initiative and package. That is a really great piece of work.
Second, may I ask you to clean-up the requirements? I was unlucky to run `pip3 install kapture`, which led to downgrading of the pytorch from 1.7 to 1.4. (?!), here is a copy-paste from install log.

> Collecting bracex>=2.0
>   Downloading bracex-2.0.1-py3-none-any.whl (10 kB)
> Installing collected packages: bracex, backrefs, wcmatch, torch, scipy, Pillow, piexif, numpy-quaternion, kapture
>   Attempting uninstall: torch
>     Found existing installation: torch 1.7.0
>     Uninstalling torch-1.7.0:
>       Successfully uninstalled torch-1.7.0
>   Attempting uninstall: scipy
>     Found existing installation: scipy 1.2.0
>     Uninstalling scipy-1.2.0:
>       Successfully uninstalled scipy-1.2.0
>   Attempting uninstall: Pillow
>     Found existing installation: Pillow 7.0.0
>     Uninstalling Pillow-7.0.0:
>       Successfully uninstalled Pillow-7.0.0

**Comment by yocabon** [comment]
Hi,

Pytorch was removed from the requirements of kapture some time ago but we didn't push a release at that time. 
It's still a dependency of kapture-localization (the matching part).
The strict 1.4.0 constraint was here because we had an error with 1.5.0 and deep-image-retrieval or r2d2 (can't remember which), but since they are not part of the repositories, I removed it (I also relaxed the numpy constraint).
Both kapture and kapture-localization master were pushed to pypi so it should no longer happen.
